src/components/NavigationMailbox.vue

```diff
@@ -107,6 +107,14 @@
 				{{ t('mail', 'Sync in background') }}
 			</ActionCheckbox>
 
+			<ActionButton 
+			    v-if="mailbox.specialRole !== 'flagged' && !account.isUnified"
+				icon="icon-brush" 
```

Comment:
Please use a material icon. The old SCSS icons are deprecated and we avoid them in new code.

This can be achieved by importing the correct icon and putting it inside the `ActionButton` like so:

```js
<ActionButton [...]>
    <template #icon>
        <CheckIcon :size="20 />
    </template>
</ActionButton>
```

and importing the icon like

```js
import CheckIcon from 'vue-material-design-icons/Check'
```

You should replace the icon `Check` with a more appropriate icon. Take a look at https://materialdesignicons.com/ for a full list.

Reply:
Here, if I understood everything correctly, I corrected the icons as you said.

Here, apparently, it needs to be done very globally, not only in this component.

It is also necessary to globally change the backlight of the delete buttons (make it red when hovering), since after using vue-material-icons `<Delete :size="20" />`, the backlight of the button does not work. Corrected in styles. But here the taste and color.

Reply:
i think i forgot this
```/**
	 * @NoAdminRequired
	 * @TrapError
	 *
	 * @param int $id
	 *
	 * @return JSONResponse
	 * @throws ClientException
	 * @throws ServiceException
	 */```
in method clerMailbox...

Reply:
Fixed